chanzuckerberg/czgenepi
# thuang-small-fixes

### Description

Write a description of your pull request here

#### Issue
https://app.clubhouse.io/genepi/story/135368/update-when-new-logo-is-ready
https://app.clubhouse.io/genepi/story/127552/by-v2-replace-favicon

### Test plan

1. Visit https://thuang-small-fixes-frontend.dev.genepi.czi.technology/data/samples
2. Log in as one of the test accounts: https://chanzuckerbergteam.slack.com/archives/C01HYJYA50W/p1620776161153400
3. You should see the new Aspen favicon and logo!
4. Also hovering over empty Lineage won't show the tooltip anymore 👌 

PTAL thank you!

<img width="1633" alt="Screen Shot 2021-05-11 at 4 52 30 PM" src="https://user-images.githubusercontent.com/6309723/117899617-59f74c00-b27c-11eb-8850-8935c33ca259.png">
